content-learn/mastering-meshery-istio/advance-concepts-of-service-mesh/istio/expose-services.mdx

```diff
@@ -1,252 +0,0 @@
----
```

Comment:
Why is this chapter being removed?

Reply:
I think I know why are you facing this issue. So, what you can do is close this PR, delete the branch. Create a new branch from the latest master code, and then add the chapter and send out a PR. Then, you won't face any more conflicts.